### ISSUE TITLE
DW::Template::Plugin::FormHTML::textbox not handling aria-describedby correctly

### ISSUE THREAD

**Issue by pauamma** [dreamwidth/dw-free#1026]
<pre>    $ret .= LJ::html_text( $args );

    foreach my $error ( @errors ) {
        $ret .= qq!<small class="error">$error->{message}</small>!;
    }

    if ( $hint ) {
        my $describedby = $args->{id} ? "$args->{id}-hint" : "";
        $ret .= qq{<span class="form-hint" id='$describedby'>$hint</span>};
        $args->{"aria-describedby"} = $describedby;
    }

    return $ret;</pre>

<code>$args->{"aria-describedby"} = $describedby;</code> comes too late to do any good. It should be up before <code>$ret .= LJ::html_text( $args );</code>
